src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -263,13 +280,13 @@ void WindowMoveHandlerPrivate::MoveSizeUpdate(HMONITOR monitor, POINT const& ptS
                 if (iter->second != m_zoneWindowMoveSize)
                 {
                     // The drag has moved to a different monitor.
-                    m_zoneWindowMoveSize->RestoreOriginalTransparency();
                     m_zoneWindowMoveSize->ClearSelectedZones();
-
                     if (!m_settings->GetSettings()->showZonesOnAllMonitors)
                     {
+                        ResetWindowTransparency(); 

```

Comment:
When moving to a different monitor, we only need to hide the zones if the `m_settings->GetSettings()->showZonesOnAllMonitors` option is off, but we don't need to reset the dragged window transparency.